config/locales/en/summary.yml

```diff
@@ -292,3 +303,30 @@ en:
         question: Details that will help us process this application
         absence_answer: *absence_none
       # END supporting evidence section
```

Comment:
I don't like the duplication of translations here and on the form. I did refactor so that these were shared; however, there are some slight differences that would require content review. This change may need to be factored into the 'partner' work when these translations are likely to become more complicated.